chainer/variable.py

```diff
@@ -434,6 +434,17 @@ def add_cand(cand):
                             x._grad += gx
             del gxs  # to reduce memory usage
 
+    def reshape(self, *shape):
+        """Returns a variable of a different shape and the same content.
+
+        .. seealso::
+           :func:`chainer.functions.reshape` for full documentation,
+
+        """
+        if len(shape) == 1 and isinstance(shape[0], tuple):
+            shape = shape[0]
```

Comment:
Could you describe why you decided to slightly change the specification of `reshape` as above?